usaon_vta_survey/templates/response/applications.html

```diff
@@ -54,31 +76,33 @@ <h3>Applications</h3>
     {% endif %}
   </div>
 
-  <h3>Add application</h3>
-  <form method="POST">
-    {{render_form(form, button_map={'submit_button': 'primary'}) }}
+  {% if not current_user.role_id=='analyst' %}
+    <h3>Add application</h3>
+    <form method="POST">
+      {{render_form(form, button_map={'submit_button': 'primary'}) }}
 
-  </form>
+    </form>
 
-  <script>
-		Highcharts.chart('highcharts-container', {
-			chart: {
-				type: 'sankey',
-			},
-			title: {
-				text: 'U.S. AON Value Tree Analysis Diagram',
-				style: {fontSize: '20px'},
-			},
-			accessibility: {
-				description: 'Sankey plot fo U.S. AON Value Tree',
-				point: {
-					valueDescriptionFormat: '{index}. {point.from} to {point.to}, {point.weight}.'
-				}
-			},
-			series: [{
-				keys: ['from', 'to', 'weight'],
-        data: {{sankey_series | safe}},
-			}]
-		});
-  </script>
+    <script>
+      Highcharts.chart('highcharts-container', {
+        chart: {
+          type: 'sankey',
+        },
+        title: {
+          text: 'U.S. AON Value Tree Analysis Diagram',
+          style: {fontSize: '20px'},
+        },
+        accessibility: {
+          description: 'Sankey plot for U.S. AON Value Tree',
```

Comment:
The hard part about accessibility is actually describing the graph contents. I have no idea how to do that :confounded: